src/Components/ArtworkAlert/Components/Steps/Filters.tsx

```diff
@@ -0,0 +1,35 @@
+import ChevronLeftIcon from "@artsy/icons/ChevronLeftIcon"
+import { Box, Clickable, Flex, Join, Separator, Text } from "@artsy/palette"
+import { FC } from "react"
+
+import { Rarity } from "Components/ArtworkAlert/Components/Filters/Rarity"
+import { useArtworkAlertContext } from "Components/ArtworkAlert/Hooks/useArtworkAlertContext"
+
+export const Filters: FC = () => {
+  const { goToDetails } = useArtworkAlertContext()
+
+  return (
+    <>
+      <Flex flexDirection="column" width="auto">
+        <Clickable
+          onClick={() => {
+            goToDetails()
+          }}
+        >
+          <Flex justifyContent="flex-start" alignItems="center" p={2}>
+            <ChevronLeftIcon />
+            <Text variant="sm">Create Alert</Text>
+          </Flex>
+        </Clickable>
+        <Separator />
+        <Box p={2}>
+          <Text variant="lg">Filters</Text>
+          <Separator my={2} />
+          <Join separator={<Separator my={2} />}>
+            <Rarity />
```

Comment:
**walkthrough**: Only adding one filter facet to simplify this already large PR. I have a branch with Ways to Buy, Medium Type, and Price filters to add in as a follow-up PR.